Makefile

```diff
@@ -1,13 +1,41 @@
-PWD         := $(shell pwd)
+MHOME         := $(shell pwd)
+PWD := $(shell pwd)
 KVERSION    := $(shell uname -r)
-KDIR        ?= /lib/modules/$(KVERSION)/build
+KDIR        := /lib/modules/$(KVERSION)/build
+DKMS       := $(shell which dkms)
```

Comment:
Redirect errors to /dev/null to avoid this:

    $ make -s clean
    which: no dkms in (/usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/usr/sbin)
    which: no dkms in (/usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/usr/sbin)


DKMS        := $(shell which dkms 2> /dev/null)